src/accountsdb/download.zig

```diff
@@ -18,7 +18,9 @@ const FullAndIncrementalManifest = sig.accounts_db.FullAndIncrementalManifest;
 
 const parallelUnpackZstdTarBall = sig.accounts_db.parallelUnpackZstdTarBall;
 
-const DOWNLOAD_PROGRESS_UPDATES_NS = 6 * std.time.ns_per_s;
+// NOTE: this also represents the amount of time a connection is given
+// to warm up before we start checking the download speed
+const DOWNLOAD_PROGRESS_UPDATES_NS = 20 * std.time.ns_per_s;
```

Comment:
Maybe we should rename this to something like "DOWNLOAD_WARMUP_TIME_NS", and then change the comment to "This also represents the interval at which progress updates are issued.".

Reply:
feels like the change would be equivalent (having the var name describe something and the comment describe its other usage) so wouldnt matter much -- unless you think the warmup time is more important to define vs progress updates?

Reply:
Yeah, I think the warmup time is a more functionally important aspect of this constant; changing it doesn't only impact the aesthetic of the output, it impacts how much warmup time we're permitting for a connection.

Reply:
agreed - 3bcc70e8